frontend/src/components/Comment/Comment.js

```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from 'react'
 import { Link } from "react-router-dom"
 import './SCSS/Comment.scss'
 import { timeElapseSincePosted } from '../../services/TimeService'
+import { EditComment } from './EditComment'
```

Comment:
i'd avoid importing it this way since it's the only function in the file. not sure if you can `export default const [name] = () => {}` but if that doesn't work, you can always just write it as `export default function [name]() {}`